src/ble/CHIPBleServiceData.h

```diff
@@ -45,11 +45,14 @@ enum chipBLEServiceDataType
  */
 struct ChipBLEDeviceIdentificationInfo
 {
-    constexpr static uint16_t kDiscriminatorMask     = 0xfff;
-    constexpr static uint8_t kAdditionalDataFlagMask = 0x1;
+    constexpr static uint16_t kDiscriminatorMask       = 0xfff;
+    constexpr static uint8_t kAdditionalDataFlagMask   = 0x1;
+    constexpr static uint8_t kAdvertisementVersionMask = 0x0f;
+    constexpr static uint8_t ShiftBits                 = 4u;
```

Comment:
Shift... for what purpose?

Reply:
We can set advertisement version using 4 bits only and in the DeviceDiscriminatorAndAdvVersion[1] first 4 bits are of advertisement version so to overwrite advertisement version shift is needed

Reply:
_I_ know that.  My point is that the name should reflect what this thing is for.